### ISSUE TITLE
Hoppie CPDLC message not showing correctly

### ISSUE THREAD

**Issue by henrylzy** [flybywiresim/a32nx#7064]
### Aircraft Version

Experimental

### Build info

```json
{
  "built": "2022-04-15T21:24:45+00:00",
  "ref": "experimental",
  "sha": "bcc3dc98b143baeaea19dedfcb92fc618e9dc415",
  "actor": "aguther",
  "event_name": "manual",
  "pretty_release_name": "experimental:bcc3dc98"
}
```

### Describe the bug

The controller sent me an message says 'MAINTAIN FL340', but showing up 'CONTACT FL340' instead.

The timestamp of the screenshots below are different because I wasn't flying with real time.

### Expected behavior

It should show the message as 'MAINTAIN FL340' instead of 'CONTACT FL340'

### Steps to reproduce

N/A

### References (optional)

![IGO64{6KQDBE705 HEV)QZ4](https://user-images.githubusercontent.com/25072307/163678431-191d2b35-082e-444e-9e7d-74407f3f7eb4.png)

![20220416221625](https://user-images.githubusercontent.com/25072307/163678584-a55cd2ed-562f-496b-b976-b208cd0dd0ab.png)

link to the message record from Hoppie: https://www.hoppie.nl/acars/system/message.html?msgid=7526003

**Comment by svengcz** [comment]
Will be fixed with #7065.
A CPDLC message is missing

**Comment by henrylzy** [comment]
> Will be fixed with #7065. A CPDLC message is missing

Sorry I forgot to mention another one. PDC sent from the controller was interpreted as something like 'MAINTAIN PRESENT HEADING'. I didn't take a screenshot for the message but the message log is this: 
https://www.hoppie.nl/acars/system/message.html?msgid=7524416

**Comment by svengcz** [comment]
Will be fixed with the same PR
